package.json

```diff
@@ -135,6 +135,7 @@
     "@babel/core": "patch:@babel/core@npm%3A7.23.2#~/.yarn/patches/@babel-core-npm-7.23.2-b93f586907.patch",
     "@types/react": "^16.9.53",
     "analytics-node/axios": "^0.21.2",
+    "@ledgerhq/hw-app-eth/axios": "^0.28.0",
```

Comment:
This has been added to mitigate a `yarn audit` issue

Reply:
@mikesposito do you think this can be reverted as per @legobeat recommendation ?

Reply:
That was originally added for a `yarn audit` issue that was fixed starting from ^0.26.x, but it seems like it's not needed with `@metamask/eth-ledger-bridge-keyring@3.x`

I removed it in [21fd490](https://github.com/MetaMask/metamask-extension/pull/26498/commits/21fd490689a47a85169489d9d9885f6218397a88)